packages/sdk/src/swap/types.ts

```diff
@@ -58,6 +58,8 @@ export interface CommonStatusFields extends ChainsAndAssets {
   depositChannelCreatedAt: number | undefined;
   expectedDepositAmount: string | undefined;
   depositChannelExpiryTime: number | undefined;
+  ccmGasBudget: string | undefined;
```

Comment:
i think i would bundle this into a `ccmMetadata` object to be consistent with the params of `requestDepositAddress`/`executeSwap `